src/main/java/uk/gov/ons/census/casesvc/service/CaseReceiptService.java

```diff
@@ -53,10 +55,12 @@ private void setUpRules() {
     rules.put(new Key("SPG", "U", CONT), new NoActionRequired());
   }
 
-  public void receiptCase(UacQidLink uacQidLink, EventTypeDTO causeEventType) {
+  public void receiptCase(UacQidLink uacQidLink, EventDTO causeEvent) {
     Case caze = uacQidLink.getCaze();
 
-    if (caze.isReceiptReceived() || uacQidLink.isBlankQuestionnaire()) return;
+    if (caze.isReceiptReceived()
+        || (uacQidLink.isBlankQuestionnaire() && !causeEvent.getChannel().equals(EQ_EVENT_CHANNEL)))
+      return;
```

Comment:
I was thinking about this, and I think we want to record the case event against the UAC-QID, even if we don't do anything else.